controllers/slides.js

```diff
@@ -23,7 +23,20 @@ const postSlides = async (req,res) => {
     }
 }
 
+const deleteSlides = async (req,res) => {
+    const { id } = req.params;
+    try{
+        await Testimonial.destroy({ where: { id } });
+        res.sendStatus(200);
```

Comment:
Si no se encuentra el slide debe devolver not found

Reply:
Maxi, ya se modificaron los cambios solicitados